components/organisms/Layout.js

```diff
@@ -0,0 +1,98 @@
+import PropTypes from "prop-types";
+import Head from "next/head";
+import { ActionButton } from "../atoms/ActionButton";
+import { SearchBar } from "../atoms/SearchBar";
+import { useState } from "react";
+
+export const Layout = ({ banner, children }) => {
+  //State for Search input
+  const [searchText, setSearchText] = useState(null);
+
+  //Handler for search input
+  function handleInputChange(e) {
+    setSearchText(e.target.value);
+  }
+
+  return (
+    <>
+      <Head>
```

Comment:
Need more here. If you're handling setting the head here such as the ability to set rich preview text and images for social media cards. Otherwise best to just remove and handle this in the respective pages